web-bluetooth/scan.js

```diff
@@ -42,3 +54,16 @@ async function onButtonClick() {
     log('Argh! ' + error);
   }
 }
+
+/* Utils */
+
+const valueDataLogger = (key, valueDataView, whatData) => {
```

Comment:
'valueDataLogger' and 'whatData' where tricky for me to understand till I read the implementation.  May I suggest:

logDataView(labelOfDataSource, key, valueDataView)

called as logDataView('Service', key, valueDataView)

Reply:
Done.